k3os/scripts/version

```diff
@@ -8,19 +8,20 @@ COMMIT=$(git rev-parse --short HEAD)
 GIT_TAG=${DRONE_TAG:-$(git tag -l --contains HEAD | head -n 1)}
 
 if [[ -z "$DIRTY" && -n "$GIT_TAG" ]]; then
-    VERSION=$GIT_TAG
+    HARVESTER_VERSION=$GIT_TAG
 else
-    VERSION="$(git describe --always --tags)${DIRTY}"
+    HARVESTER_VERSION="$(git describe --always --tags)${DIRTY}"
```

Comment:
Turns out that it is always dirty given the go mod trick in the build script. Can we do dirty detection before that?